oak-segment-tar/src/main/java/org/apache/jackrabbit/oak/segment/Segment.java

```diff
@@ -94,9 +94,9 @@ public class Segment {
     static final int RECORD_ALIGN_BITS = 2; // align at the four-byte boundary
 
     /**
-     * Maximum segment size
+     * Maximum segment size (in bytes)
      */
-    static final int MAX_SEGMENT_SIZE = 1 << 18; // 256kB
+    public static final int MAX_SEGMENT_SIZE = 1 << 18; // 256 KiB
```

Comment:
Fixing the comment here to avoid misuse of SI units and being consistent with the [Oak documentation](https://jackrabbit.apache.org/oak/docs/nodestore/segment/records.html).